Cargo.toml

```diff
@@ -49,6 +49,7 @@ env_logger = "0.10"
 hex = "0.4"
 http = "0.2"
 josekit = "0.8.3"
+leb128 = "0.2"
```

Comment:
```suggestion
```
also, need to re-run `cargo check` for `Cargo.lock` to be updated

Reply:
Done